src/components/ContactsPage.css

```diff
@@ -0,0 +1,5 @@
+.half-button {
+  width: 50% !important;
```

Comment:
'!important' keyword is highly advised to be avoided. It can get you into some funny scenarios where you lose track of what is more !important, and why. Why do you have to use it here?